server/include/Player.h

```diff
@@ -54,391 +55,393 @@ struct ShootPacketNew {
 
 struct SCachedLevel
```

Comment:
Shouldn't this be renamed to `CachedLevel` ?

Reply:
Yep.  It was missed.